sdk/core/System.ClientModel/tests/Convenience/ClientResultTests.cs

```diff
@@ -2,6 +2,7 @@
 // Licensed under the MIT License.
 
 using System.ClientModel.Primitives;
+using Azure.Core.TestFramework;
```

Comment:
Why are changes needed to ClientModel?

Reply:
Because `MockJsonModel.cs` has been moved to `Azure.Core.TestFramework` for shared usage.